src/components/display.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return <h3>{result}</h3>;
+};
```

Comment:
## Optional Change: Destructure this component 
- [ ] This `Display` component should have been restructured as inspired below. You will end up using elegant code, easy to read code and with fewer lines of code as well.
```suggestion
const Display = { (result) } => {
  return <h3>{result}</h3>;
};
```